lab-cameron/lib/list.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
```

Comment:
**TODO FOR LOST POINTS**

You need comments above each method on its Big-Oh notation.